src/LinearSolverTypes.f90

```diff
@@ -535,7 +535,7 @@ SUBROUTINE init_LinearSolverType_Base(solver,Params,A)
           CALL KSPGetPC(solver%ksp,pc,ierr)
           CALL PCSetType(pc,PCLU,iperr)
 #if ((PETSC_VERSION_MAJOR>=3) && (PETSC_VERSION_MINOR>6))
-          CALL PCFactorSetMatSolverType(pc,MATSOLVERSUPERLU_DIST,iperr)
+          CALL PCFactorSetMatSolverType(pc,MATSOLVERSUPERLU,iperr)
```

Comment:
this is something I'm carefully monitoring, but was required for the unit test to pass

Reply:
hmm... so it looks like the changed `Type` to `Package`.  But is `MATSOLVERSUPERLU` different from `MATSOLVERSUPERLU_DIST`?

Reply:
it is...the latter supports parallel SuperLU

Reply:
We dont' want that option here?

Reply:
we do, but it has issues that are unresolved (i.e. the unit test fails with it)

Reply:
Oh, gotcha.  I'm not sure I want to merge this without getting that resolved.

Reply:
only the fhp3 solver uses it, so I'm fine proceeding without this resolved, so long as we're still supporting 3.6.4

Reply:
Ok.  I could be talked into merging then.  Any idea how much effort to resolve it though?  It'd be nice to just get it done now.

Reply:
So, I'm thrown some additional time into this exclusively, but no luck.  This might be something to discuss with PETSc devs.   Looking through their fortran examples, non use SUPERLU_DIST...only SUPERLU.

Reply:
Alright.  I'm good deferring for now then.  We can revisit when we upgrade petsc.